tools/gcf_init/index.js

```diff
@@ -22,38 +22,77 @@ const datastore = Datastore();
 
 
 /**
- * Retrieves a record.
+ * Retrieves tasks given a start time, task Id or Request Id.
  *
  * @example
- * gcloud beta functions call gettask \
- *     --data '{"kind":"Task","id":"abcd1234"}'
+ * gcloud beta functions call gettasks \
+ *     --data '{"instance": "turbinia-prod", "kind":"TurbiniaTask",
+ *              "task_id":"abcd1234"}'
  *
  * @param {object} req Cloud Function request context.
  * @param {object} req.body The request body.
- * @param {string} req.body.kind The Datastore kind of the data to retrieve, e.g. "TurbiniaTask".
- * @param {string} req.body.id Id of task to retrieve
+ * @param {string} req.body.kind The kind of Datastore Entity to request
+ * @param {string} req.body.start A date string in ISO 8601 format for the start
```

Comment:
I don;t know where this is set, but I would prefer the name be something like "start_time" or similar to be a bit more descriptive.

Reply:
Done.